strongbox-commons/src/main/java/org/carlspring/strongbox/util/FileUtils.java

```diff
@@ -1,38 +1,38 @@
-package org.carlspring.strongbox.util;

```

Comment:
Are there any actual changes to the file, apart from the change of the file permissions?

Reply:
No there aren't any.It wasn't on the list , and from what i saw it doen't need any changes.
I don't know why the file permissions have been changed